src/shared/components/views/DatatableWithToggle.tsx

```diff
@@ -1,15 +1,20 @@
-import { useState, useRef } from 'react';
+import { useState, useRef, useEffect, FC } from 'react';
 import { Button } from 'franklin-sites';
-import { frame } from 'timing-functions';
+import { useParams } from 'react-router-dom';
 
 import useCustomElement from '../../hooks/useCustomElement';
 
+import { gtagFn } from '../../utils/logging';
+
 import './styles/datatable-toggle.scss';
 
-const DatatableWithToggle = ({ table }: { table: JSX.Element }) => {
-  const [tableState, setTableState] = useState('Expand');
-  const tableRef = useRef(null);
-  const tableContainerRef = useRef<HTMLDivElement>(null);
+const DatatableWithToggle: FC = ({ children }) => {
```

Comment:
A slight API change to this component, putting the table as a child of this component, because it is effectively just wrapping it